apps/web/src/components/GetStarted/ResourceGrid/ResourceGrid.tsx

```diff
@@ -0,0 +1,23 @@
+import { gridClasses } from '../../../../app/(base-org)/getstarted/page';
+import { Icon } from '../../Icon/Icon';
+import { ResourceGridProps } from '../resourceTypes';
+import ResourceCard from './ResourceCard';
+
+export default async function ResourceGrid({ section }: ResourceGridProps) {
+  return (
+    <div className={gridClasses}>
+      {section.cards.map((card, index) => (
+        <ResourceCard
+          key={card.href}
+          title={card.title}
+          description={card.description}
+          href={card.href}
+          topLeft={<span className="font-mono">{String(index + 1).padStart(2, '0')}</span>}
+          topRight={<Icon name="diagonalUpArrow" width="16px" height="16px" />}
+          colorOne={section.colorOne}
+          colorTwo={section.colorTwo}
+        />
+      ))}
+    </div>
+  );
+}
```

Comment:
imo this should prob not be nested in its own component

Reply:
removed the ResourceGrid component in favor of shared class